tests/testthat/test-library_call_linter.R

```diff
@@ -1,46 +1,39 @@
 test_that("library_call_linter skips allowed usages", {
   linter <- library_call_linter()
 
-  expect_lint(
+  expect_no_lint(
     trim_some("
       library(dplyr)
       print('test')
     "),
-    NULL,
     linter
   )
 
-  expect_lint("print('test')",
```

Comment:
current (or maybe recent?) {styler} wants this argument on the next line, so I just switched to `expect_no_lint()`, making this part of #2737